Localization/Resources/af-ZA/winget.resw

```diff
@@ -534,7 +534,7 @@ Dit kan deur die instellinglêer ‘winget settings’ gekonfigureer word.</valu
   </data>
   <data name="SourceListField" xml:space="preserve">
     <value>Veld</value>
-    <comment>The name of a peice of information about a source.</comment>
+    <comment>The name of a piece of information about a source.</comment>
```

Comment:
It's possible I should drop the localization comment changes (I never know what the policy is)

Reply:
Certainly the localization comments can be updated; they will just be overwritten by future changes if needed.